# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/sysstat/ioconf-max_blkdev.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ioconf.c.orig	2004-11-24 15:55:02.129278032 +0100
+++ ./ioconf.c	2004-11-24 15:55:12.279734928 +0100
@@ -37,6 +37,8 @@
 #define _(string) (string)
 #endif
 
+#define MAX_BLKDEV     255
+
 static unsigned int ioc_parsed = 0;
 static struct ioc_entry *ioconf[MAX_BLKDEV+1];
 
